reliable_queue.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.files = Dir['*.{md,txt}', 'lib/**/*.rb']
   s.test_files = Dir['spec/**/*.rb']
 
-  s.add_dependency 'redis', '~> 3.3'
+  s.add_dependency 'redis', '~> 4.4'
```

Comment:
shall we allow also people who are still (for whatever reason) in the 3.x?

```ruby
s.add_dependency 'redis', '>= 3.0.0'
```

Reply:
I was thinking about it. Is it not risky though? If they release redis version not compatible at some point we might be in trouble right? Or am I to cautious ?

Reply:
Hmm not sure what's the best practice here. I guess we could walk the cautious path and:

```ruby
s.add_dependency 'redis', '>= 3.0.0', '< 5.0.0'
```

Reply:
Me neither but this feels like the best solution. I will apply it.